src/libraries/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```
@@ -6,6 +6,7 @@
     <DefineConstants>$(DefineConstants);SYSNETHTTP_NO_OPENSSL;HTTP3</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <Scenario>WasmTestOnBrowser</Scenario>
```

I think we should have a condition here so it only applies to scenario to browser:
```suggestion
    <Scenario Condition="'$(TargetsBrowser)'=='true'">WasmTestOnBrowser</Scenario>
```

---

done